pkg/convert/pot_test.go

```diff
@@ -114,7 +120,7 @@ msgstr "Au revoir!"
 `),
 		},
 		{
-			name: "When msgid is multiline but with single line",
+			name: "msgid is multiline but with single line",
```

Comment:
```suggestion
			name: "single msgid with newline",
```
more concise == better